main/src/com/google/refine/exporters/CustomizableTabularExporterUtilities.java

```diff
@@ -74,13 +74,35 @@ abstract public class CustomizableTabularExporterUtilities {
 
     final static private String fullIso8601 = "yyyy-MM-dd'T'HH:mm:ss'Z'";
 
+    @Deprecated(since = "3.9")
     static public void exportRows(
             final Project project,
             final Engine engine,
             Properties params,
             final TabularSerializer serializer) {
+        exportRows(project, engine, params.getProperty("options"), serializer);
+    }
+
+    /**
+     * Export a set of rows determined by the given engine configuration to the TabularSerializer using the given
+     * options.
+     *
+     * @param project
+     *            project to export the rows from
+     * @param engine
+     *            faceted browsing configuration to filter the rows
+     * @param optionsString
+     *            JSON options dictionary serialized as a string
+     * @param serializer
+     *            a serializer which implements the TabularSerializer interface
+     * @since 3.9
+     */
+    static public void exportRows(
+            final Project project,
+            final Engine engine,
+            String optionsString,
+            final TabularSerializer serializer) {
 
-        String optionsString = (params != null) ? params.getProperty("options") : null;
```

Comment:
@wetneb when you say 

> I have doubts that using `String optionsString` is much better than using `Properties`, and so whether the round of deprecation and migration is worth it.

Is this what you're referring to? (It would help me follow along if Github's contextual commenting were used, perhaps with specific suggestions.) 

To clarify, I didn't introduce `optionString` - it's preexisting.  The old API used a `Properties` with a serialized JSON string in the "options" key. Since that was the only thing used, I simplified the API to just pass the string instead of a Map<String, String> with the serialized JSON keyed by "options" like was done for the Properties.

If you feel strongly that a 1:1 replacement with Map<String, String> is better, I'm happy to make the change, but I want to make sure I understand what your objection is.

Reply:
Ah yes, that's fine, thanks for the clarification.